schema/shared/src/test/scala/zio/blocks/schema/SchemaAspectSpec.scala

```diff
@@ -0,0 +1,48 @@
+package zio.blocks.schema
+
+import zio.Scope
+import zio.test.Assertion._
+import zio.test._
+
+object SchemaAspectSpec extends ZIOSpecDefault {
+
+  final case class Person(name: String, age: Int)
+
+  object Person extends CompanionOptics[Person] {
+    implicit val schema: Schema[Person] = Schema.derived
+
+    val nameLens: Lens[Person, String] = field(_.name)
```

Comment:
Not sure whats going on with the CI, it complains "not found: value field" here, but `field` is coming from `CompanionOptics` parent. Locally eerything passes fine.